matchbox/resource_group_test.go

```diff
@@ -17,12 +17,12 @@ func TestResourceGroup(t *testing.T) {
 	hcl := `
 		resource "matchbox_group" "default" {
  			name    = "default"
-  			profile = "foo"
-  			selector {
-				  qux = "baz"
+  	  profile = "foo"
```

Comment:
FYI, this re-formatting looks a bit strange.

Reply:
Ah, thanks. I should have left well enough alone, its a tabs vs spacing thing. Only thing needed for this PR is preferring the explicit map assignments (which will be required in v0.12)